plugins/woocommerce/src/Blocks/BlockTypes/ProductFilterCheckboxList.php

```diff
@@ -71,7 +71,7 @@ function ( $item ) {
 		ob_start();
 		?>
 		<div <?php echo get_block_wrapper_attributes( $wrapper_attributes ); // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped ?>>
-			<ul class="wc-block-product-filter-checkbox-list__list">
+			<ul class="wc-block-product-filter-checkbox-list__list" aria-label="<?php echo esc_attr( 'Filter Options', 'woocommerce' ); ?>">
```

Comment:
This should be `esc_attr__()` or remove the echo and use `esc_attr_e()`

Reply:
I already fixed it in [9b632cb](https://github.com/woocommerce/woocommerce/pull/51096/commits/9b632cb5184e30b990f29691819c035ef05444f8).